xml/snmp_sysdescr.xml

```diff
@@ -2322,6 +2322,21 @@ Copyright (c) 1995-2005 by Cisco Systems
     <param pos="1" name="os.version"/>
   </fingerprint>
 
+  <!--======================================================================
+                            Fortinet FortiGate
+   =======================================================================-->
+
+  <fingerprint pattern="^[\w\-]+\s+v([\d\.]+),build\d+,\d+(?:\s+\(\w+\))?$">
+    <description>FortiGate</description>
+    <example os.version="6.2.2">FortiaGate-61F v6.2.2,build6083,19102</example>
+    <example os.version="6.2.2">FortiaGate-61F v6.2.2,build6083,19102 (GA)</example>
+    <param pos="0" name="os.vendor" value="Fortinet"/>
```

Comment:
`FortiGate` is not an operating system. The NVD CPE dictionary has plenty of hardware entries (`h:fortinet:fortigate*`), but no operating system entries for FortiGate. Please reference existing FortiGate fingerprints [^1][^2] for guidance.

[^1]: https://github.com/rapid7/recog/blob/910f994ef4d9629abcb1339e2c4a7b2d23a69d92/xml/html_title.xml#L491-L502
[^2]: https://github.com/rapid7/recog/blob/7bc755b2f29b692befa15a2f1a991e55229f2b18/xml/telnet_banners.xml#L787-L799